lib/internal/child_process.js

```diff
@@ -692,7 +692,7 @@ function setupChannel(target, channel, serializationMode) {
     const obj = handleConversion[message.type];
 
     // Update simultaneous accepts on Windows
-    if (process.platform === 'win32') {
+    if (handle.setSimultaneousAccepts && process.platform === 'win32') {
```

Comment:
Does `setSimultaneousAccepts` exist on platforms besides Windows? If not, the `win32` check could be dropped completely.

Reply:
Currently, it is only defined on win32. Have updated !